packages/ember-application/lib/system/application.js

```diff
@@ -1003,6 +1004,7 @@ Application.reopenClass({
 
     registry.injection('view', 'renderer', 'renderer:-dom');
     registry.register('view:select', SelectView);
+    registry.register('view:outlet', OutletView);
```

Comment:
My concern is that adding this here removes the ability for apps to use the word "outlet" as a route name.

As far as why the others are not prefixed, I only see three other views being registered:

* `select` was added intentionally as public API (to be used as `{{view 'select'}}`), and was therefore not prefixed.
* `view:toplevel` and `view:default` have existed since v1.0.0-rc1 (see [here](https://github.com/emberjs/ember.js/blob/v1.0.0-rc.1/packages/ember-routing/lib/system/router.js#L59-60)).  I am unsure if it is considered public API, but it is grandfathered in under SemVer (as far as naming collisions go) due to being present pre-1.0.0.